# First and second order formation statistics

std_msgs/Header header

int32 agent_id

FormationStatistics stats